### ISSUE TITLE
ImageField and FileField transforms are incomplete

### ISSUE THREAD

**Issue by enoren** [PyCQA/pylint-django#12]
Currently only url and name attributes are being exposed to pylint through the transforms, however since under the covers these Fields are really instances of FieldFile and ImageFieldFile at runtime there are many more attributes that are needed such as the File operations, etc.

To resolve this locally I did the following within the django_db_models_fields_files.py transform definition which I think should be suitable as a solution here without having to enumerate all attributes exhaustively:

``` python
try:
    from django.db.models.fields.files import FieldFile, ImageFieldFile
except ImportError:
    FieldFile = object
    ImageFieldFile = object


class FileField(FieldFile):
    pass


class ImageField(ImageFieldFile):
    pass
```
